modules/apps/object/object-web/src/test/java/com/liferay/object/web/internal/object/entries/display/context/ObjectEntryDisplayContextImplTest.java

```diff
@@ -95,6 +96,34 @@ public void testAddFieldsetDDMFormField() {
 		Assert.assertEquals(label, localizedValue.getString(LocaleUtil.US));
 	}
 
+	@Test
+	public void testGetDefaultLocale() {
+		ObjectEntryDisplayContextImpl objectEntryDisplayContextImpl =
+			_createObjectEntryDisplayContextImpl(_mockHttpServletRequest);
+
+		_mockObjectDefinitionScope(ObjectDefinitionConstants.SCOPE_COMPANY);
+
+		LocaleUtil.setDefault(
+			LocaleUtil.BRAZIL.getLanguage(), LocaleUtil.BRAZIL.getCountry(),
+			LocaleUtil.BRAZIL.getVariant());
+
+		Assert.assertEquals(
+			LocaleUtil.BRAZIL,
+			ReflectionTestUtil.invoke(
+				objectEntryDisplayContextImpl, "_getDefaultLocale",
+				new Class<?>[] {ObjectEntry.class, Locale.class},
+				new Object[] {null, LocaleUtil.US}));
```

Comment:
@pedro-oliveira446 You can make the method "protected" to avoid using reflection.